features/support/debugging.rb

```diff
@@ -1,7 +1,6 @@
 After do |scenario|
   if scenario.failed?
     # When a scenario fails, stop running tests and open the current page.
-    save_and_open_page
```

Comment:
:( now cucumber won't automatically open your failing test in the browser for you!